src/components/Pages/Opportunities/Opportunities.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+import Project from '../../Project/Project'
+import TagCluster from '../../TagCluster/TagCluster'
+import { sortByTags, filterBySearch } from '../../../services/utils'
+import Search from '../../Search/Search'
+import Overlay from '../../Project/ProjectOverlay/Overlay'
+import Header from '../../Header/Header'
+
+export default function Opportunities({ onBack }) {
+  const [tags, setTags] = useState(null)
```

Comment:
Normally we start with an empty array, not null. That way the codes does not break at start, if we use a map somewhere.